### PR TITLE
[Ide] Draw focus rect for ImageView

### PR DESCRIPTION
**Pull request by bratsche**
Because this is an app-paintable widget it doesn't draw a focus
rectangle as needed. There doesn't seem to be a themable color for
drawing focus rectangles so I'm choosing this style color just
because it seems to look the best.

Fixes VSTS #751980

### CODE REVIEW

**Comment by slluis** [comment]
Can you please attach a screenshot of how it looks like?

**Comment by bratsche** [comment]
screen shot 2019-01-09 at 11 56 17 am

**Comment by bratsche** [comment]
screen shot 2019-01-09 at 11 57 05 am

**Comment by Therzok** [comment]
It looks misaligned? Why the `-1` on height/width for the frame height/width?

**Comment by bratsche** [comment]
screen shot 2019-01-09 at 8 59 13 pm

Here's an updated screenshot @Therzok

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Components/ImageView.cs [hunk 1]
```diff
@@ -132,7 +132,7 @@ protected override bool OnExposeEvent (Gdk.EventExpose evnt)
 					if (HasFocus) {
 						ctx.SetSourceColor (Styles.BaseSelectionBackgroundColor.ToCairoColor ());
 						ctx.LineWidth = 1.0;
-						ctx.Rectangle (x, y, image.Width - 1, Image.Height - 1);
+						ctx.Rectangle (x - Xalign, y - Yalign, image.Width + Xalign, Image.Height + Yalign);
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Components/ImageView.cs]: Using Xalign/Yalign here makes no sense, the alignment is already applied when calculating the x and y coordinates above.

**Comment by Therzok** [comment]
@bratsche still looks misaligned even in the new screenshot.

**Comment by vancura** [comment]
Can you try a thicker line? `2pt` or even `3pt`? Preferably with rounded corners, like what Quartz does.

**Comment by bratsche** [comment]
screen shot 2019-01-10 at 10 01 21 am

@vancura

**Comment by vancura** [comment]
Wow, that would be so good if it was not clipped from left and right :(

No way, please set it to the old value. Sorry!

**Comment by bratsche** [comment]
Closed this because the correct solution was to just not use ImageView anymore, instead use InformationPopoverWidget.

**Pull request closed without merging**